### PR TITLE
simple summary stats

### PR DESCRIPTION
**Pull request by hackmod**
- extract siteNotes as a directive and support simpleSummaryStats directive.

image
(screen shot with a different blockchain)

### CODE REVIEW

File: public/js/controllers/HomeController.js [hunk 1]
```diff
@@ -78,5 +78,36 @@ angular.module('BlocksApp').controller('HomeController', function($rootScope, $s
...
+.directive('simpleSummaryStats', function($http) {
+  return {
+    restrict: 'E',
+    templateUrl: '/views/simple-summary-stats.html',
+    scope: true,
+    link: function(scope, elem, attrs){
+      scope.stats = {};
+
+      var statsURL = "/stats";
+      scope.stats.ethDiff = "...";
```
Comment (hackmod) [inline public/js/controllers/HomeController.js]: oops!
this is a quick fix and have some garbage code...

File: public/views/home.html [hunk 2]
```diff
@@ -1,15 +1,6 @@
-<summary-stats></summary-stats>
-<div class="row">
-    <div class="col-md-12 offset-md-1">
-        <div class="alert alert-info">
-            <p> Now you can view and verify contract source code (<a href="/addr/0x40ebf2d6e998a76a848c41908733b26e04adffe2#tab_addr_3">example</a>). For more info about our progress check out
-                <a href="https://github.com/ethereumproject/explorer" target="_blank"> the official github respository</a>. 
-            </p><br>
-            <p>This block explorer is hosted by <a href="http://etc.minertopia.org" target="_blank">etc.minertopia.org</a>
```
Comment (hackmod) [inline public/views/home.html]: outdated link.

File: public/js/controllers/HomeController.js (hunk 1 above)
Comment (pyskell) [reply public/js/controllers/HomeController.js]: That's fine, another user is working on linking up all the stats.

**Comment by pyskell** [comment]
Merging into dev for the time being.

**Comment by hackmod** [comment]
sorry about that.

the latest block height is `docs[0].number`
~~~.diff
index af08a74..5d4f94d 100644
--- a/routes/stats.js
+++ b/routes/stats.js
@@ -52,7 +52,7 @@ var getHashrate = function(res) {
   hashFind.exec(function (err, docs) {
     var x = docs.reduce( function(hashR, doc) {
                             return { "blockTime": hashR.blockTime + doc.blockTime,
-                                     "blockHeight": doc.number,
+                                     "blockHeight": docs[0].number,
                                      "difficulty": Math.max(hashR.difficulty, doc.difficulty) }
                                  }, {"blockTime": 0, "blockHeight": 0, "difficulty": 0});
     var hashrate = x.difficulty / (x.blockTime / docs.length);
~~~

**Pull request merged**